src/i18n/strings/en_EN.json

```diff
@@ -3091,6 +3091,8 @@
     "Your message wasn't sent because this homeserver has hit its Monthly Active User Limit. Please <a>contact your service administrator</a> to continue using the service.": "Your message wasn't sent because this homeserver has hit its Monthly Active User Limit. Please <a>contact your service administrator</a> to continue using the service.",
     "Your message wasn't sent because this homeserver has been blocked by its administrator. Please <a>contact your service administrator</a> to continue using the service.": "Your message wasn't sent because this homeserver has been blocked by its administrator. Please <a>contact your service administrator</a> to continue using the service.",
     "Your message wasn't sent because this homeserver has exceeded a resource limit. Please <a>contact your service administrator</a> to continue using the service.": "Your message wasn't sent because this homeserver has exceeded a resource limit. Please <a>contact your service administrator</a> to continue using the service.",
+    "Your message wasn't sent. Your account is suspended: you cannot send messages, invites or join new rooms. To learn more, visit <detailsLink>this page</detailsLink>.": "Your message wasn't sent. Your account is suspended: you cannot send messages, invites or join new rooms. To learn more, visit <detailsLink>this page</detailsLink>.",
+    "Your message wasn't sent. Your account is suspended: you cannot send messages, invites or join new rooms. Please contact the administrator of %(homeserverDomain)s.": "Your message wasn't sent. Your account is suspended: you cannot send messages, invites or join new rooms. Please contact the administrator of %(homeserverDomain)s.",
```

Comment:
for a future review iteration: design will need to be consulted for the message copy here.

Reply:
Who exactly?

Reply:
can either get copy review internally from Write Club, or we can request design review once this hits a later stage of code review.

Reply:
Who's/what's Write Club?

Reply:
Search the corp space :)